config/docker/values.yml

```diff
@@ -1,15 +1,15 @@
 # producer format configuration
-kafka-producer.bootstrap.servers: localhost:9092
-kafka-producer.schema.registry.url: http://localhost:8081
-kafka-producer.keyFormat: binary
+kafka-producer.bootstrap.servers: broker:29092
```

Comment:
Is this 9092?

Reply:
no we need change it to 29092. That is the major reason it doesn't work before. the listener is on 29092 for docker container:

      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://broker:29092,PLAINTEXT_HOST://localhost:9092

Reply:
Cool. Thanks.